drivers/SmartThings/zwave-switch/src/aeotec-smart-switch-7/init.lua

```diff
@@ -0,0 +1,122 @@
+-- Copyright 2023 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local capabilities = require "st.capabilities"
+local Basic = (require "st.zwave.CommandClass.Basic")({ version=1 })
+local SwitchBinary = (require "st.zwave.CommandClass.SwitchBinary")({ version=1 })
+local Meter = (require "st.zwave.CommandClass.Meter")({ version = 3 })
+--- @type st.zwave.CommandClass
+local cc = require "st.zwave.CommandClass"
+
+local LAST_REPORT_TIME = "LAST_REPORT_TIME"
+local POWER_UNIT_WATT = "W"
+local ENERGY_UNIT_KWH = "kWh"
+
+local FINGERPRINTS = {
+  { mfr = 0x0086, prod = 0x0003, model = 0x00AF, deviceProfile = 'aeotec-smart-switch-7-eu' }, -- EU
+  { mfr = 0x0086, prod = 0x0103, model = 0x0017, deviceProfile = 'aeotec-smart-switch-7-us' }  -- US
+}
+
+local function can_handle(opts, driver, device, ...)
+  for _, fingerprint in ipairs(FINGERPRINTS) do
+    if device:id_match(fingerprint.mfr, nil, fingerprint.prodId) then
```

Comment:
this is the reason the tests are all failing now

your `FINGERPRINTS` map has entries with `mfr`, `prod`, and `model`, which do not match what you've used here.